OneCommunityGlobal/HighestGoodNetworkApp
# Fix the CountDownTimer component layout

# Description

![image](https://user-images.githubusercontent.com/101591708/229661246-fad47190-cfd1-4178-bd7f-415e04eb4cde.png)


## Mainly changes explained:

At the BadgeReport component, a span tag was mistakenly applied globally instead of being intended for another component (UserProfile) where it was meant to be applied.
I deleted this global span tag and will open another PR to fix the bug that it was intend to fix at UserProfile component.


## How to test:
check into current branch
do `npm install` and `...` to run this PR locally
log as any user
go to dashboard


## Screenshots or videos of changes:
 
**Before** 

https://user-images.githubusercontent.com/101591708/229664566-329eb1fa-471b-4c36-992d-d8b4771ebd9d.MP4


**After**

https://user-images.githubusercontent.com/101591708/229664579-488139fb-d319-469c-8718-1f86e88d97a1.MP4
